package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wp-to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly I can not run your project because you miss putting the `Start` script, please fix it 🙏🏼  
`[HINT]`
```json
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open"
```

![image](https://github.com/MenendezON/WP-To-Do-list/assets/63915024/96bf189e-a861-4242-aed9-7075a8d62c74)